docs/en/user/user-documentation.md

```diff
@@ -0,0 +1,92 @@
+* If **Show in Detail** is checked, this field will be shown in the detail view that opens when a visitor clicks a row on the table shown to them. This is enabled for all fields by default.
+* If **Only show one value if duplicates exist** removes one of two (or more) results that have the same data in the result set shown to the website visitor
+* **Presented order** allows an editor to change the ordering of columns with more precision than the drag and drop provided on the GridField. E.g. it is possible to move a column between page of a GridField
+* The **Result conditions** allow for a "pre-filter" to be chosen to only show results matching (or not matching) a particular value.
+
+## Filters:
+The filters tab allows a CMS content editor to configure fields shown to a visitor that will allow them to search the results shown to them. One is always created by default after selecting a resource and saving the page. This default filter is a text type that searches all columns for matches to the term the visitor enters.
+
+By default there are two types of filters available to a content editor to configure, text and dropdown. The text filter allows a user to enter free text to filter results for. Dropdown type only allows pre-defined values to be chosen from.
+
+### Configuring a text filter:
+
+![Configuration options for a text filter](textFilterConfig.png)
+
+* A **Filter  label** can be given to indicate what type of search the content editor configures it to do.
+* **Column sources** allow the filter to be configured to either search either all columns, or if the checkbox is unchecked to search only a particular column or set of columns (multiple may e chosen).
+
+### Configuring the dropdown filter:
+
+This filter type has only one extra configuration from those provided by the text filter; the ability to supply options for the user to choose from.
+
+#### Options
+
+After choosing which columns the filter will apply to, the options selection will update to provide a list of all possible unique values those columns contain. The content editor can then choose to disable any of those options by un-checking the checkbox.
+
+![Dropdown filter configuration](dropdownFilterConfig.png)
+
+#### Delimiter
+
+If these options are grouped, e.g. with the _Family Services Directory_ we can see a _`SERVICE_TARGET_AUDIENCES`_ column - any row for this column may contain multiple values separated by a comma, e.g. "Children,Youth"
+
+To get these values to display as two options (both "Children", and "Youth") we can enter a comma into the delimiter box - the list of options to will update to be separated appropriately.
+
+![Dropdown filter configuration using Delimiter](delimiterUse.png)
+
+#### Manual selection
+
+The Manually add options entry box for this allows values to be entered directly by the content editor, one option per line.
```

Comment:
```suggestion
The "Manually add options" entry box for this allows values to be entered directly by the content editor, one option per line.
```

Reply:
🤔 I've used **bold** to indicate UI options elsewhere in the document, should this also be **bold** rather than "quoted"?

Reply:
Went with bold for consistency.

Reply:
Please don't resolve things like this - it makes them disappear. The writing guide is [here](https://silverstripe.atlassian.net/wiki/spaces/PROD/pages/520290326/Writing+product+user+help+or+guides) (see "Referencing titles, names, locations or actions:") and says that standalone UI elements (e.g. Security files admin area) should be italicised, while UI buttons should be in quotes.

I'll revert your bolding

Reply:
Sorry for the confusion everyone, that was our fault with the style guide since it's very new. I've updated it and have pushed a new commit adding the correct styling.